C++/Navio/Common/RCOutput.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RCOUTPUT
+#define RCOUTPUT
+
+#define PWN_OUTPUT 0
+
+
+class RCOutput
+{
+public:
+    virtual bool initialize() = 0;
+    virtual void setFrequency(float frequency) = 0;
```

Comment:
setFrequency should work on a channel basis.

And also stick to underscores in naming.

It should be rather set_frequency.